StripeFinancialConnections/StripeFinancialConnections/Source/Native/Shared/AutoResizableUIView.swift

```diff
@@ -0,0 +1,81 @@
+//
+//  AutoResizableUIView.swift
+//  StripeFinancialConnections
+//
+//  Created by Krisjanis Gaidis on 7/11/24.
+//
+
+import Foundation
+import SwiftUI
+import UIKit
+
+//
+// These are only used for SwiftUI previews so no need to expose outside of DEBUG.
+//
+#if DEBUG
+
+/// This helps to auto-resize UIView's placed in SwiftUI. Otherwise,
+/// the UIView's tend to stretch the full height of the screen.
+///
+/// If wrapping the `UIView` with `AutoResizableUIView` doesn't help,
+/// also call `applyAutoResizableUIViewModifier` to your SwiftUI view.
+class AutoResizableUIView<ContentView: UIView>: UIView {
```

Comment:
this logic can mostly be ignored - it's for SwiftUI previews; though will be useful in the future if we ever start bridging SwiftUI to UIKit